rclcpp/test/test_logging.cpp

```diff
@@ -163,6 +164,39 @@ TEST_F(TestLoggingMacros, test_logging_skipfirst) {
   }
 }
 
+TEST_F(TestLoggingMacros, test_throttle) {
+  using namespace std::chrono_literals;
+  rclcpp::Clock steady_clock(RCL_STEADY_TIME);
+  for (uint64_t i = 0; i < 3; ++i) {
+    RCLCPP_DEBUG_THROTTLE(g_logger, steady_clock, 10000, "Throttling");
+  }
+  EXPECT_EQ(1u, g_log_calls);
+  RCLCPP_DEBUG_SKIPFIRST_THROTTLE(g_logger, steady_clock, 1, "Skip first throttling");
+  EXPECT_EQ(1u, g_log_calls);
+  for (uint64_t i = 0; i < 3; ++i) {
+    RCLCPP_DEBUG_THROTTLE(g_logger, steady_clock, 10, "Throttling");
+    std::this_thread::sleep_for(10ms);
```

Comment:
@BMarchi shouldn't we call the macro more than 3 times in that same time lapse to verify throttling?

Reply:
I think it still applies. Sleeping for 5ms and duplicating the amount of iterations form 3 to 6 would prove the point.